tools/src/main/java/com/nedap/archie/rmobjectvalidator/validations/RMMultiplicityValidation.java

```diff
@@ -21,7 +21,8 @@ public static List<RMObjectValidationMessage> validate(CAttribute attribute, Str
             if (cardinality != null) {
                 if (!cardinality.getInterval().has(collectionValue.size())) {
                     String message = RMObjectValidationMessageIds.rm_CARDINALITY_MISMATCH.getMessage(cardinality.getInterval().toString());
-                    return Lists.newArrayList(new RMObjectValidationMessage(attribute, pathSoFar, message));
+                    RMObjectValidationMessageType messageType = cardinality.getInterval().isMandatory() ? RMObjectValidationMessageType.REQUIRED : RMObjectValidationMessageType.DEFAULT;
```

Comment:
I think it could be better if this is a separate message type. Required means that a field is required, and is not present in the data. 
this means a cardinality mismatch is present, meaning that an attribute has X values, which is not within the constraint y..z.
These will have to be displayed differently in forms: required will often be a short message or icon near a single field. Cardinality mismatch will be a message, icon or otherwise on a group of fields. 
These message types are useful in things such as forms, to show a correct message and/or UI element. Because required and cardinality mismatch are different concepts, and should be shown differently as well, I think a different message type is better than using required here.